portfolio/src/main/webapp/index.html

```diff
@@ -10,8 +10,41 @@
         <div id="content">
             <h1>My Portfolio</h1>
             <h2>This is Ankita's portfolio.</h2>
+
+            <p2>
+                <div style="text-align:center;padding:1em 0;"> <h4>
+                    <a style="text-decoration:none;" href="https://www.zeitverschiebung.net/en/city/5368361">
+                    <span style="color:gray;">Current local time in</span><br />Los Angeles, United States</a></h4> 
+                    <iframe id = "time" src="https://www.zeitverschiebung.net/clock-widget-iframe-v2?language=en&size=small&timezone=America%2FLos_Angeles" 
+                    width="100%" height="90" frameborder="0" seamless></iframe> </div>
+                    <br><br><br><br>
+                    <textarea name=”VisitorComment” rows=”20″ cols=”20″>Take notes here!</textarea><br><br><br>
+
+                    My favorite places: <br>(click to visit!)<br><br>
+                    <a target="_blank" href="http://www.udessertstorysf.com/berkeley.html">
+                        <img src="/images/food.jpg" alt="Food picture!" style="width:150px">
+                    </a> <br> <br><br>
+                    <a target="_blank" href="https://www.isoladiburano.it/en/">
+                        <img src="/images/burano.jpg" alt="Burano, Italy" style="width:250px">
+                    </a><br> <br><br>
+                    <a target="_blank" href="http://www.streettacony.com/">
+                        <img src="/images/taco.jpg" alt="Street taco!" style="width:150px">
+                    </a><br> <br><br>
+                    <a target="_blank" href="https://www.visitbushkillfalls.com/">
+                        <img src="/images/bushkill.png" alt="Food picture!" style="width:250px">
+                    </a>
+            </p2>
+
             <iframe src="https://open.spotify.com/embed/playlist/3PujhiyIMnr3HhCmoLMdB9" width="300" height="380" 
             frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>
+
+            <p3>
+                <iframe id = "time" width="266" height="150" src="https://www.youtube.com/embed/nHuI_SnqlWE" frameborder="0" 
```

Comment:
Do you want to use a different frame "id" than time. This may require you to add corresponding css style as well.

Using same "id" for two independent elements could result in undefined behavior, for example, when using with javascript:
> document.getElementById("#time")

Reply:
Makes sense - updated.